core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,27 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.reloadDummyExplorations = function(){
+          // Create a dummy exploration with random title.
+          AdminTaskManagerService.startTask();
+          $scope.setStatusMessage('Processing...');
+          for (var i = 0; i < $scope.DummyExplorationCount; ++i) {
+            var title = ['Hulk Neuroscience',
+                         'Quantum Starks',
+                         'Wonder Anatomy',
+                         'Elvish, language of "Lord of the Rings',
+                         'The Science of Superheroes'
+            ];
+            var randomTitle = Math.floor( Math.random() * title.length);
+            $http.post('/contributehandler/create_new', {
+              title: title[randomTitle],
+            }).then(function(response) {
+              $scope.setStatusMessage('Dummy explorations reloaded.');
```

Comment:
Made few changes here. Please take a look and see if it is what you meant in the above comments.